src/components/Filter/FilterInput.vue

```diff
@@ -354,17 +291,86 @@ export default {
       }
     },
   },
+  methods: {
+    /**
+     * Focuses the input
+     * @returns {Promise<void>}
+     */
+    async focusInput() {
+      // make sure everything is rendered
+      await this.$nextTick();
+      // focus the input
+      this.$refs.input.focus();
+
+      if (!this.input && this.resetActiveTags) {
+        this.resetActiveTags();
+      }
+    },
+    async resetFilters(hideTags = false) {
+      this.setFilterInput('');
+      this.setSelectedTags([]);
+
+      if (!hideTags) {
+        // We prevent blur from hiding tags
+        this.$emit('update:preventedBlur', true);
+        if (this.resetActiveTags) {
+          this.resetActiveTags();
+        }
+        await this.focusInput();
+        return;
+      }
+
+      this.resetedTagsViaDeleteButton = true;
+      this.showSuggestedTags = false;
+      this.$refs.input.blur();
+    },
+    focusFirstTag() {
+      // make sure we show the suggestedTags, in case we lost focus
+      if (!this.showSuggestedTags) {
+        this.showSuggestedTags = true;
+      }
+      // make sure that the suggestedTags ref exists
+      if (this.hasSuggestedTags && this.$refs.suggestedTags) {
+        this.$refs.suggestedTags.focusFirstTag();
+      }
+    },
+    setFilterInput(value) {
+      this.$emit('input', value);
+    },
+    setSelectedTags(tags) {
+      this.$emit('update:selectedTags', tags);
+    },
+    deleteTags(array) {
+      this.setSelectedTags(this.selectedTags.filter(tag => !array.includes(tag)));
+    },
+    async handleBlur(event) {
+      // if the blur came from clicking a link
+      const target = event.relatedTarget;
+      if (target && target.matches && target.matches('button, input, ul')) return;
+      // Wait for mousedown to send event listeners
+      await this.$nextTick();
+
+      this.resetActiveTags();
+
+      if (this.preventedBlur) {
+        this.$emit('update:preventedBlur', false);
+        return;
+      }
+      this.showSuggestedTags = false;
+    },
+  },
 };
 </script>
 
 <style scoped lang="scss">
 @import 'docc-render/styles/_core.scss';
 
-$filter-bar-background-color: rgba(255, 255, 255, .8);
-$tag-outline-padding: 4px;
-$input-vertical-padding: 13px;
+$tag-outline-padding: 4px !default;
+$input-vertical-padding: rem(13px) !default;
 
 .filter {
+  --input-vertical-padding: #{$input-vertical-padding};
```

Comment:
this should allow for easy height adjustment of the input, when used from other components.